rest/models/DashboardTemplate.go

```diff
@@ -54,23 +54,24 @@ type GridItem struct {
 	Height    int    `json:"h"`
 	MaxHeight int    `json:"maxH"`
 	MinHeight int    `json:"minH"`
+	Static    bool   `json:"static"`
```

Comment:
FWIW this is part of react-grid-view's Layout type here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-grid-layout/index.d.ts#L66. It is marked optional boolean.

I'm proposing we add it to the server tracked struct as without it the UI could only have (on initial load of session - removing localStorage from the equation):
* All widgets locked by default
* All widgets unlocked by default

However with this struct change we can also have:
* Some widgets locked and some widgets unlocked by default

By default as of now, the service defaults `static` to `false`.